docs/data/material/components/dialogs/dialogs.md

```diff
@@ -153,7 +153,7 @@ Follow the [Modal performance section](/material-ui/react-modal/#performance).
 
 Follow the [Modal limitations section](/material-ui/react-modal/#limitations).
 
-## Complementary projects
+## Supplementary projects
```

Comment:
Sync with the new header used by Sam in #44191. I guess it makes more sense https://ell.stackexchange.com/questions/188457/difference-between-complementary-and-supplementary.